_posts/2018-06-20-microsoft-extensions-configuration-deep-dive.md

```diff
@@ -418,7 +418,7 @@ builder.AddJsonFile("appsettings.json")
 
 The builder doesn't actually invoke any of the things that read configuration. It just gives you an opportunity to specify your configuration _sources_. It also has a `Properties` dictionary on it that you can use when people register sources. For example, if you wanted to ensure a particular source only gets registered one time with a `ConfigurationBuilder`, you could synchronize on that `Properties` dictionary.
 
-Each time you call one of the extensions (`AddJsonFile`, `AddXmlFile`) that adds an `IConfigurationSource` to the `ConfigurationBuilder`. The `ConfigurationBuilder` keeps track of these until you tell it to `Build()`.
+Each time you call one of the extensions (`AddJsonFile`, `AddXmlFile`), it adds an `IConfigurationSource` to the `ConfigurationBuilder`. The `ConfigurationBuilder` keeps track of these until you tell it to `Build()`.
```

Comment:
The original version sounded like "that" was introducing a restrictive clause, and I kept looking for a payoff at the end of the sentence.